Neos.Neos/Resources/Private/Templates/Module/Management/Workspaces/Show.html

```diff
@@ -180,15 +203,36 @@
 		<div class="neos-pull-right">
 			<f:if condition="{canPublishToBaseWorkspace}">
 				<f:then>
-					<button type="submit" name="moduleArguments[action]" value="publish" class="neos-button neos-hidden neos-disabled batch-action" disabled="disabled">{neos:backend.translate(id: 'workspaces.publishSelectedChanges', source: 'Modules', package: 'Neos.Neos')}</button>
-					<button type="submit" name="moduleArguments[action]" value="discard" class="neos-button neos-hidden neos-disabled batch-action" disabled="disabled">{neos:backend.translate(id: 'workspaces.discardSelectedChanges', source: 'Modules', package: 'Neos.Neos')}</button>
-					<button class="neos-button neos-button-danger" data-toggle="modal" href="#discard">{neos:backend.translate(id: 'workspaces.discardAllChanges', source: 'Modules', package: 'Neos.Neos')}</button>
-					<button form="postHelper" formaction="{f:uri.action(action: 'publishWorkspace', arguments: {workspace: selectedWorkspace})}" type="submit" class="neos-button neos-button-primary">{neos:backend.translate(id: 'workspaces.publishAllChangesTo', source: 'Modules', package: 'Neos.Neos', arguments: {0: baseWorkspaceLabel})}</button>
+					<button class="review-button-action neos-button neos-button-danger" data-toggle="modal" href="#discard">
```

Comment:
A `button` has no `href`-Attribut. Additionally `type="button"` should be set because if not, it will always act as submit button. By reading I asume that should not happen here.

Reply:
The `href` is for js to know which modal to open, which is required when using `data-toggle="modal"` afaik.

Reply:
This answers the question why it's here. But it still is invalid HTML and we should avoid introducing something like this in the future.